app/templates/js/index.jsx

```diff
@@ -1,6 +1,6 @@
 /*** @jsx React.DOM */
 
 var React = require('react'),
-    App = require('./components/App');
+    App = require('./components/App.jsx');
```

Comment:
Does this cause issues for you? Just curious because it doesn't for me. Want to make sure I'm not missing something.